### ISSUE TITLE
"indention" wiktionary says synonym

### ISSUE THREAD

**Issue by polluks** [codespell-project/codespell#3154]
https://github.com/codespell-project/codespell/blob/6c3294016b04b594671e0e1704e096c0eec9504d/codespell_lib/data/dictionary_rare.txt#L124
https://en.wiktionary.org/wiki/indention

**Comment by DimitriPapadopoulos** [comment]
We rely on dictionaries and indeed most of them accept both:
* [OED](https://www.oed.com/search/dictionary/?q=indention)
* [Oxford Learner's](https://www.oxfordlearnersdictionaries.com/spellcheck/english/?q=indention)
* [Cambridge](https://dictionary.cambridge.org/dictionary/english/hanging-indent?q=indention)
* [Collins](https://www.collinsdictionary.com/dictionary/english/indention)
* [Longman](https://www.ldoceonline.com/spellcheck/english/?q=indention)
* [Merriam-Webster](https://www.merriam-webster.com/dictionary/indention)

More importantly, I believe dictionaries here should be aligned with the open-source reference for spell checking:
* [SCOWL and Friends](http://app.aspell.net/lookup?dict=en_US-large;words=indention).

In short, a merge request is welcome :smile:

**Comment by DimitriPapadopoulos** [comment]
True, but reviewers can apply the rule of manually looking up [SCOWLS (and Friends)](http://app.aspell.net/lookup?dict=en_US-large;words=indention) until a mechanism is in place.

**Comment by polluks** [comment]
Rare seems to be okay...
